### ISSUE TITLE
Podfile update for cocoapods 1.0.1

### ISSUE THREAD

**Issue by s00500** [jgh-/VideoCore#323]
Hi,
Using cocoapods 1.0.1 you need to specify targets for the pod dependencys in the podfile

Like so:
`target "SampleBroadcaster-Swift" do
 pod 'VideoCore', path: '../..'
end`

Otherwise yo get the error: 
[!] The dependency `VideoCore (from`../..`)` is not used in any concrete target.

**Comment by ankur-sardar** [comment]
@s00500: Is this code running? I am facing so many errors during the compilation in Xcode

**Comment by omarojo** [comment]
@s00500 can you elaborate more on this ? what is exactly what your are describing ?

**Comment by s00500** [comment]
@omarojo The podfile needs to be upgraded for newer versions of cocoapods

**Comment by bintu1234** [comment]
yes i am also facing same issue,before i am installed with cocoapods 0.38.2 but now it showing like need to update cocoapods  in terminal,so i just updated to new version. Now i am getting this issue `'type_half.inl' file not found`.can any body help me please
